<?xml version="1.0" ?>

<launch>
  
   <arg name="robot_id" default="robot_1"/>
   <arg name="simulate" default="true"/>
   <param name="use_sim_time" value="false"/>

  
   <!-- BEGIN ROBOT "robot_id" --> 
  <group ns="$(arg robot_id)">
    <param name="tf_prefix" value="$(arg robot_id)" />
    
    	 <node name="waiter_agent" pkg="tb_waiter" type="waiterAgentDra" output="screen">
    	    <param name="simulation"  value="true"/>
    	    <param name="robot_name"  value="$(arg robot_id)"/>
    	    <param name="enable_mark" value="false"/>
    	 </node>
   
  </group>
    
</launch>
